src/python/pants/backend/terraform/tool.py

```diff
@@ -217,6 +219,19 @@ def default_known_versions(cls):
             "1.0.6|linux_arm64|2047f8afc7d0d7b645a0422181ba3fe47b3547c4fe658f95eebeb872752ec129|30514636",
         ]
 
+    extra_env_vars = StrListOption(
+        help=softwrap(
+            """
+            Additional environment variables that would be made available to all Terraform processes.
+            """
+        ),
+        advanced=True,
+    )
+
+    @property
+    def env_vars(self) -> tuple[str, ...]:
+        return tuple(sorted(set(self.extra_env_vars)))
```

Comment:
For this kind of cases in which we want to have an accessor property to an option, we normally follow the pattern of prefixing the option variable with a `_` so it's private and then we can use the same name of the option as the accessor property. Something like:

```python
_extra_env_vars = StrListOption(...)

@memoized_property
def extra_env_vars(self) -> tuple[str, ...]:
    return tuple(sorted(set(self._extra_env_vars)))
```

Preserving the same name in the accessor property helps identifying as the option we want to access.

Reply:
That makes sense. I think I looked at a more complicated example. I've simplified it the way the Helm backend does it.